doc/whats_new/v1.1.rst

```diff
@@ -28,12 +28,29 @@ parameters, may produce different models from the previous version. This often
 occurs due to changes in the modelling logic (bug fixes or enhancements), or in
 random sampling procedures.
 
+:mod:`sklearn.cluster`
+......................
+
```

Comment:
We can remove this since we normally do not include the section header in the "Change Models" section.